packages/dx-grid-core/src/plugins/table-band-header/computeds.ts

```diff
@@ -73,3 +75,67 @@ export const tableHeaderColumnChainsWithBands: GetHeaderColumnChainsFn<
 
   return [...bandChains, ...chains.slice(maxBandRowIndex)];
 };
+
+const getBandLevel: PureComputed<[any[], string, number?], number> = (
+  bands, bandTitle, level = 0,
+) => {
+  for (const band of bands) {
+    if (band.title === bandTitle) {
+      return level;
+    }
+    if (band.children !== undefined) {
+      const result = getBandLevel(band.children, bandTitle, level + 1);
+      if (result >= 0) return result;
```

Comment:
We can include `result >= 0` into above `if`

Reply:
I can't see how. Can you please expand your idea?

Reply:
My bed, you are right